### ISSUE TITLE
"actSig" nil error

### ISSUE THREAD

**Issue by dhleong** [ray-x/lsp_signature.nvim#145]
<img width="1197" alt="Screen Shot 2021-12-21 at 1 08 49 PM" src="https://user-images.githubusercontent.com/816150/146978005-1a1bd064-509b-4bd1-b95b-284695d9b4cd.png">

Repro'ing for me with tsserver

**Comment by ray-x** [comment]
It seems to be an old version.
Would you check the latest one? If the issue still existed, please send me the function signature as well. Thanks!

**Comment by dhleong** [comment]
Huh! I was sure I did a `PlugUpdate` and tried again before posting but now I can't reproduce that error.

I did run into this while impatiently waiting for tsserver to spin up, but may not be related to this plugin:

<img width="369" alt="Screen Shot 2021-12-21 at 5 17 57 PM" src="https://user-images.githubusercontent.com/816150/147004927-3e3829f8-287a-4106-82c5-370d2732818e.png">

The signatures don't show super consistently with tsserver, but that could just be tsserver's fault; rust_analyzer seems pretty quick and responsive.

The signature in question was from zustand, for what it's worth:

<img width="966" alt="Screen Shot 2021-12-21 at 5 22 08 PM" src="https://user-images.githubusercontent.com/816150/147005644-3942c577-b4a3-4635-a89c-b7d96246dff3.png">
